contracts/api-spec.yaml

```diff
@@ -613,6 +650,82 @@ components:
           format: int64
           example: '2'
 
+    StreamStatModel:
+      description: A stat that is related to a specific stream
+      properties:
+        id:
+          type: integer
+          format: int64
```

Comment:
Should we instead have nested properties like the json output of srt-live-transmit? If that's too much of a pain to store in the database, can we reformat at the service layer or something?
![image](https://user-images.githubusercontent.com/32281260/110163025-83e44c00-7dbd-11eb-82d8-2c409d9b55f7.png)

Reply:
given that i save all the attributes in one table, i think it would add a bit of boiler plate code if i would to do them as seprate objects

Reply:
let me know if you  have idea

Reply:
i think maybe i can just create nested objects or something

Reply:
Your idea was good, i applied it. Now it should be good

Reply:
Works great now! 😄